core/plugin_api.php

```diff
@@ -51,14 +51,31 @@
 require_api( 'lang_api.php' );
 require_api( 'logging_api.php' );
 
-# Cache variables #####
-
+/**
+ * Plugins cache
+ * @global MantisPlugin[] $g_plugin_cache
```

Comment:
Now that you are adding better documentation to these variables, i'd find useful to specify the index and value structure
Such as making clear that this array is indexed by the plugin base names.
i don't know the best syntax for that. An example i think i have used before, similar to:
` array( 'plugin_basename' => MantisPlugin, ... )`

Reply:
OK, will do